### PR TITLE
Added recipe for nudup NuGEN deduplication tool

### PR DESCRIPTION
**Pull request by lparsons**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/nudup/bld.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+md "bin"
+copy "nudup.py" "%PREFIX%\bin"
```
Comment (bgruening) [inline recipes/nudup/bld.bat]: We do not support windows so we can ignore these files :)

File: recipes/nudup/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  name: nudup
+  version: "2.2_post2016104"
+
+source:
+  git_rev: 740d9fe439dd8917605a56483a8796b377eb24c6
+  git_url: https://github.com/nugentechnologies/nudup.git
+
+build:
+  number: 0
+  skip: True # [win]
```
Comment (bgruening) [inline recipes/nudup/meta.yaml]: [win or py3k]

Comment (lparsons) [inline recipes/nudup/meta.yaml]: I wondered about that, but didn't see it in the docs. Thx.

File: recipes/nudup/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  skip: True # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - samtools >=0.1.18
```
Comment (lparsons) [inline recipes/nudup/meta.yaml]: I looked for that in the docs, but couldn't find it. Thanks for pointing it out.

**Comment by lparsons** [comment]
@bioconda/core please review

**Pull request merged**